src/app/shared/services/apis/gb3/gb3-geoshop-bounding-box.service.ts

```diff
@@ -0,0 +1,39 @@
+import {Injectable} from '@angular/core';
+import {Gb3ApiService} from './gb3-api.service';
+import {Observable} from 'rxjs';
+import {BboxGeoshop} from '../../../models/gb3-api-generated.interfaces';
+import {ApiGeojsonGeometryToGb3ConverterUtils} from '../../../utils/api-geojson-geometry-to-gb3-converter.utils';
+import {map} from 'rxjs/operators';
+import {HasBoundingBox} from '../../../interfaces/has-bounding-box.interface';
+
+type GeoshopBbox = 'ZH' | 'CH';
```

Comment:
One last idea:

```
type GeoshopBbox = 'ZH' | 'CH';

const BboxEndpoints: Record<GeoshopBbox, string> = {
  ZH: 'canton',
  CH: 'switzerland',
} as const;
```

What this will will do is tie it to the GeoshopBbox type. In that way, they are in sync, in that they key values will always be identical to the other ones. Also, "ZH" instead of "canton" seems preferrable, if more cantons are added (_could_ be, if data set could be ordered that exceeds canton borders).

But you can also leave it as is :)